File: lib/layer2_core/light/e_on_error_container_light.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/light/system.dart';
import '../../layer1_raw/light/standard.dart';
import '../../layer1_raw/light/blue.dart';
import '../../layer1_raw/light/green.dart';
import '../../layer1_raw/light/red.dart';
import '../../layer1_raw/light/nothing.dart';

class EOnErrorContainerLight {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyl44;
      case 'standard': return cstl44;
      case 'blue': return cbl44;
      case 'green': return cgl44;
      case 'red': return crl44;
      case 'nothing': return cnol44;
      default: return cstl44;
    }
  }
}

File: android/src/main/kotlin/com/example/design_engine/layer2_core/light/EOnErrorContainerLight.kt
package com.example.design_engine.layer2_core.light
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object EOnErrorContainerLight {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyl44
            AppTheme.STANDARD -> R.color.cstl44
            AppTheme.BLUE -> R.color.cbl44
            AppTheme.GREEN -> R.color.cgl44
            AppTheme.RED -> R.color.crl44
            AppTheme.NOTHING -> R.color.cnol44
        }
    }
}
